content/en/Platform Deep Dive/Credits/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Cobalt Credits"
+linkTitle: "Credits"
+weight: 40
+description: >
+  We offer a flexible pricing model.
```

Comment:
You probably can convince me otherwise, but I cringe when I see "pricing" in product docs (2 places). I know it's from the Website...

```suggestion
  We offer a flexible consumption model.
```